### ISSUE TITLE
Can you use this with RaspberryPi

### ISSUE THREAD

**Issue by tobiasmuecksch** [nRF24/RF24Audio#18]
Is it possible to use this repo on a raspberry pi?
Can you explain how to?

**Comment by TMRh20** [comment]
No, but you can use the core RF24 lib to stream wav files to Arduinos running this repo. 

You need 8bit, 16khz, mono wav files and need to send them in 32 byte chunks. The radio addressing also needs to match.
